Services/Migration/DBUpdate_3136/classes/class.ilDBUpdate3136.php

```diff
@@ -1,5 +1,18 @@
-<?php
-/* Copyright (c) 1998-2010 ILIAS open source, Extended GPL, see docs/LICENSE */
+<?php declare(strict_types=1);
+
+/******************************************************************************
```

Comment:
According to our `Rector` in 'CI/Rector/ChangeLicenseHeader.php' the license header has the wrong format:

```php
/**
 * This file is part of ILIAS, a powerful learning management system
 * published by ILIAS open source e-Learning e.V.
 *
 * ILIAS is licensed with the GPL-3.0,
 * see https://www.gnu.org/licenses/gpl-3.0.en.html
 * You should have received a copy of said license along with the
 * source code, too.
 *
 * If this is not the case or you just want to try ILIAS, you'll find
 * us at:
 * https://www.ilias.de
 * https://github.com/ILIAS-eLearning
 *
 *********************************************************************/
```

Reply:
Thank you for the hint. I didn't know where to pick it. Hopefully now it is correct.